### ISSUE TITLE
Add Day-Night Theme 

### ISSUE THREAD

**Issue by beinganonymous** [PatilShreyas/GDGPune-DevFest19-Android#19]
Add Day-Night Theme which can be adjusted by button click and TimeZone

**Comment by PatilShreyas** [comment]
Hi @beinganonymous. 
We have implemented Material Design library in our app. Stable `DayNight` theme is currently available in AppCompat support library. In Material Design library, this theme is available but not in stable release, it's available in beta version. What you think? What will have to do? Should we use beta library?

**Comment by beinganonymous** [comment]
Hi @PatilShreyas.
I think we should try out beta version of `DayNight` theme and lets see how it goes.
Or we can design 2 base application theme,
1. Theme.MaterialComponents.Light
2. Theme.MaterialComponents.Dark

and we can change these themes using buttons and according to scenario.
What do you say ?

**Comment by PatilShreyas** [comment]
Yes right. Then we will give try to beta version first. If it works well then fine otherwise your suggestion will be helpful. Thanks 😃
